src/service/server.rs

```diff
@@ -113,24 +101,6 @@ fn route_status(inbound: TcpStream, config: Arc<Config>, server: Arc<Server>, pe
     tokio::spawn(service);
 }
 
-/// Route inbound TCP stream to proxy, spawning a new task.
-#[inline]
-fn route_proxy(inbound: TcpStream, config: Arc<Config>) {
```

Comment:
Where did this go?

Reply:
Same as https://github.com/timvisee/lazymc/pull/42#discussion_r1102727835

Was only used to directly route client to server without parsing any packets if the server has started, now we have (potentially) multiple servers so we need to parse the handshake packet first and keep the packet queue